formula/nwchem.rb

```diff
@@ -0,0 +1,70 @@
+class Nwchem < Formula
+  desc "NWChem: Open Source High-Performance Computational Chemistry"
+  homepage "http://www.nwchem-sw.org"
+  url "https://github.com/nwchemgit/nwchem/releases/download/6.8.1-release/nwchem-6.8.1-release.revision-v6.8-133-ge032219-src.2018-06-14.tar.bz2"
+  version "6.8.1"
+  sha256 "23ce8241a5977a93d8224f66433851c81a08ad58a4c551858ae031485b095ab7"
+
+  depends_on "gcc" # for gfortran
+  depends_on "open-mpi"
+  depends_on "scalapack"
+  depends_on "openblas" => :recommended
+
+  def install
+    pkgshare.install "QA"
+
+    cd "src" do
+      (prefix/"etc").mkdir
+      (prefix/"etc/nwchemrc").write <<~EOS
```

Comment:
Doesn't upstream provide a reasonable default configuration file?

Reply:
No, it does not. There are several issues to be addressed and this has not happened yet. What I have introduced in this formula is very similar to the solution adopted by the packager of some Linux distributions